DataFormats/TotemRPDetId/interface/TotRPDetId.h

```diff
@@ -0,0 +1,186 @@
+/****************************************************************************
+ *
+ * This is a part of TOTEM offline software.
+ * Authors: 
+ *	Hubert Niewiadomski
+ *	Jan Kašpar (jan.kaspar@gmail.com) 
+ *
+ ****************************************************************************/
+
+#ifndef TotRPDetId_h
```

Comment:
please keep the package name in the include guards
```  DataFormats_TotemRPDetId_TotRPDetId ```

I don't see a big reason to abbreviate. TotemRPDetId is more intuitively clear